### ISSUE TITLE
Changing the properties of an individual bullet after time

### ISSUE THREAD

**Issue by Horikaze** [Moonzel/Godot-PerfBullets#14]
Hey, is it possible to change the direction of an individual bullet after a given time? For example, changing Vector2 after 2s
![exa](https://github.com/Moonzel/Godot-PerfBullets/assets/87659929/a0014416-c2ef-47a5-afcf-3032747c1225)

**Comment by savakujundzic** [comment]
What about just adding a spawner there, and after 2s you stop the original spawner and run the second spawner that has bullets in that direction?

**Comment by Horikaze** [comment]
Yes, this is one way to get around this problem, I wonder if there is another more convenient way. For example, a script in BulletType that changes the behavior of the bullet.

**Comment by Moonzel** [comment]
Because of the way the Bullets are created and behave, this is not something I would recommend doing. If you did want to adjust this, you could potentially be able to change the direction property of the Bulprops resource that that bullet uses, but it is resource intensive to recursively change something per bullet in that way.
